internal/pass/sync/handlers.go

```diff
@@ -26,8 +26,67 @@ func ExtensionRequestSync(syncingApp *Syncing) gin.HandlerFunc {
 			return
 		}
 
+		resp, err := syncingApp.RequestSync(gCtx, fcmToken)
+		if err != nil {
+			logging.Errorf("Failed to request sync: %v", err)
```

Comment:
yyou can use log from context here as well and everywhere in that handler

Reply:
done, thanks